src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -939,16 +939,10 @@ protected AssemblySymbol PrimaryDependency
         }
 
         /// <summary>
-        /// Return error code that has highest priority while calculating use site error for this symbol. 
+        /// Returns true if the error code is the highest priority while calculating use site error for this symbol. 
         /// Supposed to be ErrorCode, but it causes inconsistent accessibility error.
         /// </summary>
-        protected virtual int HighestPriorityUseSiteError
-        {
-            get
-            {
-                return int.MaxValue;
-            }
-        }
+        protected virtual bool IsHighestPriorityUseSiteErrorCode(int code) => true;
```

Comment:
>true [](http://example.com/codeflow?start=78&length=4)

Should we return ```false``` instead, in order to align with the old behavior?

Reply:
The old behavior accepted the first error encountered, as `HighestPriorityUseSiteError` was `Int32.MaxValue` by default. This replicates that behavior.

Reply:
Note that VB and C# differ in behavior here. C# accepted the first error, VB did not.